src/socket/tcp.rs

```diff
@@ -474,7 +489,7 @@ const DEFAULT_MSS: usize = 536;
 
 impl<'a> Socket<'a> {
     #[allow(unused_comparisons)] // small usize platforms always pass rx_capacity check
-    /// Create a socket using the given buffers.
+    /// Create a socket using the given buffers
```

Comment:
This seems to have been changed by mistake.

Reply:
I will fix. Thank you.

Reply:
fixed